cif_core.dic

```diff
@@ -16876,6 +16876,30 @@ save_DATABASE_CODE
 
 save_
 
+save_database_code.bincstrdb
+
+    _definition.id                '_database_code.BIncStrDB'
+
+    loop_
+      _alias.definition_id
+         '_database_code_BIncStrDB'
+         '_database.code_BIncStrDB'
```

Comment:
If I understand correctly, `_database.*` aliases were introduced to account for these data names existing in the mmCIF/PDBx dictionary. However, since that dictionary has now deprecated the `database` category in favour of the new `database_2` category, no newly defined `_database_code.*` items require the `_database.code_*` alias (`database_2` category uses a completely different mechanism based on enumeration lists). I recommend removing this alias.